ftlib/commlib/nccl/Dockerfile.cn

```diff
@@ -0,0 +1,24 @@
+FROM dockerhub.azk8s.cn/nvidia/cuda:10.1-devel-ubuntu18.04
+
+RUN sed -i "s@http://.*archive.ubuntu.com@http://mirrors.huaweicloud.com@g" /etc/apt/sources.list && \
```

Comment:
Are these really necessary? It seems hacky.

Reply:
It just convenient for me to build the image in CN. That's why I put `.cn` to the Dockerfile name. But if that seems undesired, it can be removed.